.github/PRERELEASE_NOTES.md

```diff
@@ -0,0 +1,38 @@
+Here you will find the build artifacts for the last successful build of the master branch of the Plasma repository. This builds are automatically generated and are not tested. They may include bugs, errors, and partially-completed work.  
+**These are made available for you to use at your own risk.**
+
+For Age building and development activities, use an **Internal** client.  
+For playing Uru on public servers, use an **External** client.
```

Comment:
I would remove this line, because AFAIK there are currently no public shards where you can safely play with a mainline H'uru client. Even if there was one, you would still need an internal client to use `/LocalData` and prevent the client from "updating" itself to the standard client provided by the server.

```suggestion
```

Reply:
Agreed. External clients are currently only useful for shard operators.

Reply:
Do we want to archive them here then? I guess maybe it's still useful as a place for shard operators to grab them

Reply:
Yeah. I think we still want to have them archived for anyone who wants to set up their own complete shard.